### ISSUE TITLE
Add alias of 'result'

### ISSUE THREAD

**Issue by cmeeren** [fsprojects/FSharpPlus#325]
I use both FSharpPlus and FsToolkit.ErrorHandling in all my projects. The latter defines a `result` builder, which conflicts with the `result` operator from FSharpPlus. AFAIK there are other standard names for the `result` operator (not entirely sure – `pure`? `retn`?). I suggest FSharpPlus adds one or more of these standard aliases for `result`.

**Comment by wallymathieu** [comment]
You could add your own operator in your own code by noting that the result operator is simply:
https://github.com/fsprojects/FSharpPlus/blob/3fdc13372d05c388e8e3f3f9d53ee197230bf526/src/FSharpPlus/Operators.fs#L104-L105

**Comment by cmeeren** [comment]
Yep, I have, I just figured that it'd be nice to have in FSharpPlus too, since both FSharpPlus and FsToolkit.ErrorHandling are very useful libraries that work nicely together.

**Comment by gusty** [comment]
Just to add more information, `pure` is unfortunately a reserved word in F#, `retn` is an abbreviation and abbreviations are not encouraged in the F# guidelines.

I think we could add a unary operator but not sure which.

**Comment by cmeeren** [comment]
> I think we could add a unary operator but not sure which.

But that doesn't work well with piping, right? E.g. instead of

```f#
a |> b |> c |> result
```

you would have to write

```f#
<op>(a |> b |> c)
```

right?

In any case, I get the point about abbreviations, though I think it can work if the abbreviation is clear and if it is for a good reason (e.g. to avoid a reserved keyword).

**Comment by gusty** [comment]
I would rather say:

> you would have to write:

    a |> b |> c |> (op)

Any suggestions for either operator or abbreviation?
